components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -29,44 +29,42 @@ A shared file system is always required to create **OpenSearch** snapshots. To r
 
 ### Setting up the backup configuration
 
--   Create an EFS file system, please refer sample steps [here](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html)
+- Create an EFS file system, please refer sample steps [here](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html)
+
+- We have to mount EFS to all the vm's manually. To do that please refer [this](https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-helper-ec2-linux.html) 
 
 #### Configuration in OpenSearch Node
 
--   Mount the shared file system on all OpenSearch and Frontend servers:
+- Mount the EFS on all OpenSearch Node. For example you mount the EFS to folder structure `/mnt/automate_backups/`
 
-    ```sh
-    mount /mnt/automate_backups
-    ```
-
--   Create an OpenSearch sub-directory and set permissions to one of the OpenSearch servers (only if the network mount is correctly mounted).
+- Create an OpenSearch sub-directory and set permissions to one of the OpenSearch servers (only if the network mount is correctly mounted).
 
     ```sh
-    sudo mkdir /mnt/automate_backups/opensearch
+    sudo mkdir -p /mnt/automate_backups/opensearch
```

Comment:
Can we mention /mnt/automate_backups as 'base_path' or with someother generic terminology?
<img width="470" alt="image" src="https://user-images.githubusercontent.com/54614142/216626509-df58a0aa-891d-4c45-b95e-60747b0ffc22.png">